### PR TITLE
Update alt text for icons in news & events section

### PR DESCRIPTION
**Pull request by nick-mon1**
### Summary
Made several accessibility updates for author images on several pages.

### Problem
[Home](https://digital.gov) and [News](https://digital.gov/news/) pages have author images that aren't accessible with their `title` attributes. This approach is not consistent across all AT devices.

[Author](https://digital.gov/authors/katina-stapleton/) bio pages need better plain language text to accompany the image.

### Solution
Added a `span` with `usa-sr-only` within the link to ensure screen readers read "Posts by author name".
Updated verbiage for author bio image to read "Profile image for author name". Previously was "Photo:author name"

### Notes
Fixed a typo in the `title` which was missing a space between the name and "Posts by".

### CODE REVIEW

File: themes/digital.gov/layouts/authors/list.html [hunk 1]
```diff
@@ -9,9 +9,9 @@
           <header>
 
             {{- if .Params.github -}}
-              <img class="profile" src="https://github.com/{{- .Params.github -}}.png?size=100" srcset="https://www.github.com/{{-  .Params.github -}}.png?size=200" alt="Photo: {{- .Params.display_name -}}">
+              <img class="profile" src="https://github.com/{{- .Params.github -}}.png?size=100" srcset="https://www.github.com/{{-  .Params.github -}}.png?size=200" alt="Profile image for&nbsp;{{- .Params.display_name -}}">
```
Comment (mejiaj) [inline themes/digital.gov/layouts/authors/list.html:12]: I don't think we need the non-breaking space here.

File: themes/digital.gov/layouts/authors/list.html [hunk 2]
```diff
@@ -9,9 +9,9 @@
...
             {{- else -}}
-              <img class="profile" src="{{- "img/digit-light.png" | absURL -}}" alt="Photo: {{- .Params.display_name -}}">
+              <img class="profile" src="{{- "img/digit-light.png" | absURL -}}" alt="Profile image for&nbsp;{{- .Params.display_name -}}">
```
Comment (mejiaj) [inline themes/digital.gov/layouts/authors/list.html:14]: Same here. Usually attributes don't require a non-breaking space.

File: themes/digital.gov/layouts/partials/core/get_authors_short.html [hunk 3]
```diff
@@ -26,10 +26,10 @@
       {{/* Lets look up the author data, based on the UID. Should equal the filename */}}
       {{- with $.Site.GetPage (printf "/%s/%s" $taxonomy $uid) -}}
 
-      <a class="author" href="{{- (printf "authors/%s/" $uid) | absURL -}} " title="Posts by {{- .Params.display_name | default $uid -}} " rel="author">
+      <a class="author" href="{{- (printf "authors/%s/" $uid) | absURL -}} " title="Posts by&nbsp;{{- .Params.display_name | default $uid -}} " rel="author">
```
Comment (mejiaj) [inline themes/digital.gov/layouts/partials/core/get_authors_short.html:29]: Having the title and the `usa-sr-only` text inside could cause it to be read twice on some screen readers.

I would remove one.

**Review by mejiaj — CHANGES_REQUESTED** [review]
Hey Nick, added some suggestions for improving screen reader experience.

File: themes/digital.gov/layouts/authors/list.html (hunk 1 above)
Comment (nick-mon1) [reply themes/digital.gov/layouts/authors/list.html:12]: On my browser, the empty space is ignored. I have removed the non-breaking space.

File: themes/digital.gov/layouts/authors/list.html (hunk 2 above)
Comment (nick-mon1) [reply themes/digital.gov/layouts/authors/list.html:14]: Same note as above.

File: themes/digital.gov/layouts/partials/core/get_authors_short.html (hunk 3 above)
Comment (nick-mon1) [reply themes/digital.gov/layouts/partials/core/get_authors_short.html:29]: I have remove the `title` attribute and kept the `usa-sr-only` span.

**Review by mejiaj — APPROVED** [review]
LGTM, thanks!

**Pull request merged**